types/index.d.ts

```diff
@@ -784,6 +784,12 @@ declare namespace RamdaAdjunct {
         */
         allSettledP<T>(iterable: Iterable<T>): Promise<Array<SettledPromise<T>>>;
 
+        /**
+         * anyP accepts an iterable of promises and returns a promise that is fulfilled by the first given promise to be fulfilled,
```

Comment:
Doesn't match the description in the source: https://github.com/char0n/ramda-adjunct/pull/1080/files#diff-4cb076891bd6a121df6d7ea58b2d06b7R11. We need to have this identical.

Reply:
Thank you, I fixed it